source/concepts/protocols_funding_withdrawal.rst

```diff
@@ -0,0 +1,155 @@
+
+Withdrawal protocol
+-------------------
+
+A ledger channel must be concluded before funds can be withdrawn. For each
+asset, participants can withdraw the amount corresponding to their address in
+the balance of the concluded state back to any on-chain account. The on-chain
+account must be specified in the withdraw call and the authorization must be
+signed using the keys corresponding to the participant's off-chain address in
+the channel.
+
+The participant could also withdraw their amount in multiple parts, to
+different on-chain accounts, as long as the authorization is signed by the
+participant in each call. On each call, the withdrawn amount will be
+subtracted. Withdraw calls would succeed as long as the amount remaining is
+greater than or equal to the amount in the withdraw call.
+
+
+.. note::
+    
+   ``Withdraw from channel`` in the below diagram represents the participant
+   withdrawing their entitled amounts. This in itself could be one or more
+   transactions, withdrawing to same or different on-chain accounts.
+
+.. image:: ../_generated/concepts/settle_withdraw_ledger.svg
+  :align: Center
+  :alt: Image not available
+
+Sub-channel
+===========
+
+Funding protocol
+----------------
+
+A sub-channel is funded by sending an off-chain transaction on the parent
+ledger channel, that locks the funds corresponding to the opening balance of
+the sub-channel. This update must be sent by the participant that proposed the
+channel and the proposee must wait for it.
+
+.. image:: ../_generated/concepts/open_funding_sub.svg
+  :align: Center
+  :alt: Image not available
+
+Withdrawal protocol
+-------------------
+
+If a sub-channel was finalized through off-chain transactions, then funds can
+be withdrawing as per the final balance by sending an update on the parent
+ledger channel that unlocks the funds as per the final balance. This update
+must be sent by the participant that proposed the channel and the proposee must
+wait for it.
+
+.. image:: ../_generated/concepts/settle_withdraw_sub.svg
+  :align: Center
+  :alt: Image not available
+
+If a sub-channel was not finalized through off-chain transaction or settling it
+through off-chain update fails, then it must finalized and settled on the
+blockchain.  See :ref:`settling_on_the_blockchain` section in settle phase for
+more details.
+
+Virtual channel
+===============
+
+Funding protocol
+----------------
+
+A virtual channel is funded by sending two off-chain transactions. One
+transaction on each of the the parent ledger channels between the common
+intermediary and each of the participants.
+
+Intermediary will be notified of this update. If they accept it, the channel
+will be funded. If they reject, funding will fail.
+
+.. image:: ../_generated/concepts/open_funding_virtual.svg
+  :align: Center
+  :alt: Image not available
+
+Withdrawal protocol
+-------------------
+
+If a virtual channel was finalized through off-chain transactions, then funds
```

Comment:
What if a virtual channel cannot be finalized off-chain?

Reply:
Have included a description for this case.